anitya/lib/backends/cran.py

```diff
@@ -0,0 +1,120 @@
+# -*- coding: utf-8 -*-
+
+"""
+ (c) 2018 - Copyright Elliott Sales de Andrade
```

Comment:
Once you move your copyright claim, can you use this docblock to provide an overview of the module? Just an introduction CRAN with pointers to more detailed documentation is fine. Something similar to https://github.com/release-monitoring/anitya/blob/e5e199a7b9d88df1465a8a1041140e93a3b3f2eb/anitya/lib/backends/crates.py#L19 if possible.